docs/release_notes/v1.10.4.md

```diff
@@ -21,3 +21,21 @@ A regression was introduced in Dapr 1.10.0 that caused all proxied gRPC requests
 ### Solution
 
 We have changed how API allowlists work for gRPC, so all proxied calls are always allowed regardless of the rules in the allowlists. We have added tests to prevent future regressions.
+
+## Fixes Dapr calling the application to get subscriptions when the metadata endpoint is invoked
```

Comment:
Please add a link to this section at the top

Reply:
added